server/tests/use_cases/test_add_service_commitments.py

```diff
@@ -0,0 +1,106 @@
+"""
+test_add_service_commitments.py: Tests for the add_service_commitments use case.
+"""
+
+from use_cases.add_service_commitments import add_service_commitments
+
+if __name__ == "__main__":
+    from unittest.mock import Mock
+
+    # valid shift ID (successful case)
```

Comment:
@emrameduseljac your test cases do not run as part of our test suite because you have them hiding behind 
```
if __name__=="__main__"
```
We want these test cases to run along side other test. To run all test cases locally on your computer, in the server directory, set PYTHONPATH:
`export PYTHONPATH=$(pwd)`
Then run test cases with `pytest`
You will see a listing of all files that ran. When I run pytest on my computer, your test cases do not run. 
Remove the "main" and just include the test case functions in this file. Run the tests to verify that they pass.